jormungandr/src/rest/server/error.rs

```diff
@@ -1,38 +1,28 @@
-use native_tls::Error as TlsError;
-use std::error::Error as StdError;
-use std::fmt::{Display, Formatter, Result as FmtResult};
-use std::io::Error as IoError;
+use std::io;
+use thiserror::Error;
 
-#[derive(Debug)]
+#[derive(Debug, Error)]
 pub enum Error {
-    Pkcs12LoadFailed(IoError),
-    Pkcs12Invalid(TlsError),
-    BindFailed(IoError),
+    #[error("failed to open server certificate file")]
+    ServerCertFileOpenFailed(#[source] io::Error),
```

Comment:
All these `Server` prefixes just add typing, please remove them.

The existing members (`ServerAlreadyStopped` and below) do make some narrative sense.